core/src/main/java/org/stellar/anchor/auth/JwtService.java

```diff
@@ -136,8 +155,16 @@ public <T extends AbstractJwt> T decode(String cipher, Class<T> cls)
       return (T) Sep10Jwt.class.getConstructor(Jwt.class).newInstance(jwt);
     } else if (cls.equals(Sep24InteractiveUrlJwt.class)) {
       return (T) Sep24InteractiveUrlJwt.class.getConstructor(Jwt.class).newInstance(jwt);
-    } else {
+    } else if (cls.equals(Sep24MoreInfoUrlJwt.class)) {
       return (T) Sep24MoreInfoUrlJwt.class.getConstructor(Jwt.class).newInstance(jwt);
+    } else if (cls.equals(PlatformAuthJwt.class)) {
+      return (T) PlatformAuthJwt.class.getConstructor(Jwt.class).newInstance(jwt);
+    } else {
```

Comment:
Nit: let's explicitly check that the class is `CallbackAuthJwt` and throwing an `NotSupportedException` in the else case.

Reply:
We can add a  `@SuppressWarnings({"ConstantConditions"})` to the decode method.

Reply:
That's probably about the same as as the previous approach.